src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -157,19 +162,30 @@
 )
 
 
+# Set Keywords
+bigquery_dialect.sets("unreserved_keywords").clear()
+bigquery_dialect.sets("unreserved_keywords").update(
+    [n.strip().upper() for n in bigquery_unreserved_keywords.split("\n")]
+)
+
+bigquery_dialect.sets("reserved_keywords").clear()
+bigquery_dialect.sets("reserved_keywords").update(
+    [n.strip().upper() for n in bigquery_reserved_keywords.split("\n")]
+)
+
 # Add additional datetime units
 # https://cloud.google.com/bigquery/docs/reference/standard-sql/date_functions#extract
 bigquery_dialect.sets("datetime_units").update(
     ["MICROSECOND", "DAYOFWEEK", "ISOWEEK", "ISOYEAR", "DATE", "DATETIME", "TIME"]
 )
 
-# Unreserved Keywords
-bigquery_dialect.sets("unreserved_keywords").add("SYSTEM_TIME")
-bigquery_dialect.sets("unreserved_keywords").remove("FOR")
-bigquery_dialect.sets("unreserved_keywords").add("STRUCT")
-bigquery_dialect.sets("unreserved_keywords").add("ORDINAL")
-# Reserved Keywords
-bigquery_dialect.sets("reserved_keywords").add("FOR")
+# # Unreserved Keywords
```

Comment:
Can we remove this code rather than commenting it out?

Reply:
Oh yeah forgot about this. Left them in while testing. They are gone!